index.html

```diff
@@ -10,11 +10,13 @@
   <body>
     <div class="banner">
       <img src="./assets/images/Giant_Panda.png" />
-      <div class="ellipse"></div>
-      <p class="text header-1">
-        Watch your <span style="color: yellow">favorite</span> animal online
-      </p>
-      <button class="button button-type-1">watch online</button>
+      <div class="banner-content">
+        <div class="ellipse"></div>
+        <p class="text header-1">
```

Comment:
move p and button to one div.
make margins between button and p.
and center alignment for button.